frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -52,19 +61,31 @@ const applicationAdmins = shallowRef<FamAppAdminGetResponse[]>(
 const applicationsUserAdministers = computed(() => {
     return LoginUserState.getApplicationsUserAdministers();
 });
+const delegatedAdmins = shallowRef<FamAccessControlPrivilegeGetResponse[]>(
+    props.delegatedAdmins
+);
```

Comment:
Maybe move this new ref close to the other ref(s) above.

Reply:
just moved, thank you!